src/lib/y2partitioner/clients/main.rb

```diff
@@ -93,9 +93,15 @@ def commit
 
       # Partitioner dialog is initalized with the probed and staging devicegraphs
       #
-      # @return [Dialogs::Main]
+      # @return [Dialogs::Main, nil] nil if it was not possible to get the
+      #   devicegraphs (probing failed)
       def partitioner_dialog
-        @partitioner_dialog ||= Dialogs::Main.new(storage_manager.probed, storage_manager.staging)
+        return @partitioner_dialog if @partitioner_dialog
+
+        # Force quitting if probing failed
+        return nil unless storage_manager.probed
```

Comment:
NP: I would add a comment to explain that a popup is shown with the error during the probing, and the client is forced quitting only when the user selects abort option.